cves/CVE-2016-8747.yml

```diff
@@ -43,7 +43,7 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Apache refactored how Tomcat reads request buffers to be more performant. A bug was introduced where Tomcat may read more data from buffer than it should causing it to leak other requests data.
```

Comment:
```suggestion
description: |
	Apache refactored how Tomcat reads request buffers to be more performant. 
	A bug was introduced where Tomcat may read more data from buffer than it should 
	causing it to leak other requests data.
```

Reply:
Using the pipe character ("|") and using indented newlines makes the formatting much easier to read longer responses. (I would do this everywhere)